.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,185 @@
+## How to contribute
+
+Share your knowledge with the community by contributing to Magento DevDocs!
+You can contribute by creating an issue or pull request (PR) on our [devdocs](https://GitHub.com/magento/devdocs){:target="_blank"} GitHub repository.
+We welcome all types of contributions; from minor typo fixes to new topics.
+
+Magento's tech writer team and [Maintainers]({{ page.baseurl }}/contributor-guide/devdocs-maintainers.html) review issues and pull requests on a regular basis, and we do our best to address all issues as soon as possible.
+Working through the backlog takes time, though, so we appreciate your patience.
+
+## Rewards and thanks
+
+DevDocs works with the Community Engineering teams and projects.
+As you contribute PRs, you gain Contribution Points.
+
+If you write and contribute a full topic, we will put your name (or your company's name) at the top of the page right under the title, and link it to your blog or website!
+We track and post your picture and a link to your GitHub account on the [Top recent contributors]({{ page.baseurl }}/contributor-guide/quarterly-contributors.html) page.
+
+## Get started
+
+* Make sure you have a [GitHub account](https://github.com/signup/free).
+* Make sure you sign the [Magento Contributor Agreement](http://www.magento.com/legaldocuments/mca).
+* Check the [guidelines](#guidelines).
+* [Fork and clone](#fork) the repository.
+* Write content and commit.  Need help, see our [templates](#templates).
+* Create a [pull](#pull-request).
+
+<div class="bs-callout bs-callout-info" id="info" markdown="1">
```

Comment:
Callout boxes are not supported in GH markdown, so these tags can be removed.
To make it stand out, I suggest converting to:

``` md
**Note:**
_If you are not sure where to start contributing, you can check GitHub issues tagged with [help wanted](https://github.com/magento/devdocs/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) and [good first issue](https://github.com/magento/devdocs/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)._
```

Renders as

**Note:**
_If you are not sure where to start contributing, you can check GitHub issues tagged with [help wanted](https://github.com/magento/devdocs/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) and [good first issue](https://github.com/magento/devdocs/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)._

Reply:
Removing, thanks!